src/styles/index.scss

```diff
@@ -1,3 +1,113 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.cards {
+  --color: #00acdc;
+  --color-second: #fff;
```

Comment:
If you use scss create scss variable and use it here. Also, after a created move all variables to the separate file